sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -432,12 +452,18 @@ private void setHealthCheck(
             Protos.TaskInfo.Builder taskInfo,
             String serviceName,
             PodInstance podInstance,
-            TaskSpec taskSpec) {
+            TaskSpec taskSpec,
+            GoalStateOverride override) {
         if (!taskSpec.getHealthCheck().isPresent()) {
             LOGGER.debug("No health check defined for taskSpec: {}", taskSpec.getName());
             return;
         }
 
+        if (override.equals(GoalStateOverride.STOPPED)) {
```

Comment:
Should this be `GoalStateOverride.PAUSED`?

Reply:
ditto